models/staging/snowplow/events_base.sql

```diff
@@ -0,0 +1,19 @@
+with raw_events as (
```

Comment:
interested in understanding why there are duplicates - let's chat about this IRL